bower.json

```diff
@@ -1,10 +1,19 @@
 {
-  "name": "HabitRPG",
-  "version": "0.1.1",
```

Comment:
According to https://github.com/bower/spec/blob/master/json.md, version has been deprecated.
But if bower has a timeline itself to be deprecated in Habitica, then I will these changes out of the PR.